models/engine/file_storage.py

```diff
@@ -0,0 +1,61 @@
+"""
+    This module handles serialization/deserialization of objects
+"""
+import json
+from pathlib import Path
+
+class FileStorage:
+    """
+    Serializes instances to a JSON file and deserializes
+    JSON file to instances
+
+    """
+    def __init__(self, file_path, objects):
```

Comment:
The __objects and __file_path attributes are private class attributes and FileStorage class doesn't have an init method. In this code, line 13-16 you have made objects and file_path as instance attributes instead of class attributes

Reply:
I guess I would have to look up the differences between class attributes and instance attributes

Reply:
To make it a class attribute, you just gotta define it outside that __init__ method